server/handlers/result_handlers.py

```diff
@@ -95,6 +95,26 @@ def ready(self):
 
     def __call__(self, result_batch):
         logger.info(f"\n\nHandler received batch: {result_batch}\n\n")
+
+        # omit failed tasks for now
+        # TODO handle in DIA-1122
+        result_batch = [record for record in result_batch if not record["_adala_error"]]
+
+        # only keep the parts of the record we know how to handle
+        known_keys = [
+            # 'text',  # should always be omitted
+            "task_id",
+            "output",
+            # TODO in DIA-1122
+            # '_adala_error',
+            # '_adala_message',
+            # '_adala_details',
+        ]
+        result_batch = [
```

Comment:
seems like a good place to have pydantic types for validation, maybe?

Reply:
done